src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -507,6 +507,34 @@ def set_browser_implicit_wait(self, seconds):
         implicit_wait_in_secs = robot.utils.timestr_to_secs(seconds)
         self._current_browser().implicitly_wait(implicit_wait_in_secs)
 
+    def get_browser_info(self, type='object'):
```

Comment:
Should the keyword name be changed. Info is quite vague word and does not describe what the keyword does. 

Perhaps get_browsers_desired_capabilities is better name?